.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
I guess we can include it. I don't think this will break as the commands are pretty straight forward. I will continue powering through the PR changes which will fix the build.

Reply:
e28631a4f23a45bc0c94f54fabf0a552066b66b6

Reply:
lgtm!